doc/design/subscription-config.md

```diff
@@ -0,0 +1,50 @@
+# Subscription Config
+
+## Configuring Operators deployed by OLM
+
+It is possible to configure how OLM deploys an operator via the `config` field in the [subscription](../../pkg/api/apis/operators/subscription_types.go) object.
+
+Currently, the `SubscriptionConfig` object allows users to configure the following features on Deployments created by OLM:
+
+### Selector
+The `selector` field defines a list of `Label` selectors for pods. Existing ReplicaSets whose pods are selected by this will be the ones affected by this deployment.
```

Comment:
nit: reword the second sentence to remove passive voice?

Reply:
Resolved by https://github.com/operator-framework/operator-lifecycle-manager/pull/1058#discussion_r332242758